tools/nl/embeddings/utils.py

```diff
@@ -223,3 +223,14 @@ def save_embeddings_lancedb(local_dir: str, embeddings: List[Embedding]):
 def save_index_config(fm: FileManager, index_config: IndexConfig):
   with open(fm.index_config_path(), 'w') as f:
     yaml.dump(asdict(index_config), f)
+
+
+def get_additional_catalog_embeddings_name(additional_catalog_path: str) -> str:
```

Comment:
hmm, what is additional_catalog specific about this function?

should it be:  `get_first_index_name()`.   Also, this should probably be in admin.py?

Reply:
Renamed.

See other comment on admin/html.py.